an-apluss/epic-mail
# Revert "[finish ft-signup-164296649] Add signup page"

#### What does this PR do?
- This display a page to take user's input for registration

#### Description of Task to be completed?
- Create a user interface to accept user's input for registration
#### How should this be manually tested?
- Ensure you have access to internet
- Ensure you have web browser installed
- Click [link]( https://an-apluss.github.io/epic-mail/UI/signup.html )

#### What are the relevant pivotal tracker stories?
- PT Story link - ( `[164296649](https://www.pivotaltracker.com/story/show/164296649)`)

#### Screenshots 
![sign up page - epic - an-apluss github io](https://user-images.githubusercontent.com/45961117/53900450-db1f7c80-403c-11e9-916b-9212e15f771e.png)
